packages/components/app/styles/components/table.scss

```diff
@@ -258,4 +267,23 @@ $hds-table-cell-padding-tall: 22px 16px 21px 16px; // the 1px difference is to a
       vertical-align: baseline;
     }
   }
+
+  // multi-select (isSelectable=true)
+
+  .hds-table--is-selectable & {
+    .hds-table__th--is-selectable {
+      width: 1px; // shrink-wrap content (will expand as needed)
+      padding-right: 8px;
```

Comment:
@didoo Do you think I should create a sass variable for this padding value?

Reply:
Let's review the table cell (`hds-table__th--is-selectable`) layout today, there is something strange for me... 🤔